src/app/groups/group-member-contribution-assigner/group-member-contribution-assigner.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, Input, Inject, OnInit } from '@angular/core';
+import { gradeService, projectService, groupService} from 'src/app/ajs-upgraded-providers';
+import {GroupMemberService} from 'src/app/api/models/doubtfire-model';
+import * as _ from 'lodash';
```

Comment:
I saw many people use `lodash`, just wondering if this needed when we have [array](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array) @PerryRose, @tancnle

Reply:
> I saw many people use lodash, just wondering if this needed when we have [array](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array)

I agree with you. We don't need `lodash` in this case.  In fact, I think we can drop this dependency altogether since there is no usage.

```shell
❯ rg lodash -l --type coffeescript --type ts
src/app/common/grade-icon/grade-icon.component.ts
```

I can create a separate PR to retire `lodash` from our dependencies.

Reply:
@RayGuo-ergou PR to remove `lodash` https://github.com/thoth-tech/doubtfire-web/pull/44